aspnetcore/blazor/security/webassembly/additional-scenarios.md

```diff
@@ -267,6 +267,10 @@ A component creates the <xref:System.Net.Http.HttpClient> from the <xref:System.
 > [!NOTE]
 > The controller in the server API, `WeatherForecastNoAuthenticationController` for the preceding example, isn't marked with the [`[Authorize]`](xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute) attribute.
 
+To determine whether a secure client or an insecure client should be the default <xref:System.Net.Http.HttpClient> instance, consider the number of authenticated versus unauthenticated endpoints that the app contacts. If the majority of the app's requests are to secure API endpoints, use the authenticated <xref:System.Net.Http.HttpClient> instance as the default. Otherwise, register the unauthenticated <xref:System.Net.Http.HttpClient> instance as the default.
```

Comment:
This sounds like something we encourage people do. The reality is that this is completely up to the customers' preferences. Let's be less assertive here, and use some softer phrasing:

```suggestion
The decision whether secure client or an insecure client should be the default <xref:System.Net.Http.HttpClient> instance is completely up to the developer. One simple approach you can take is to consider the number of authenticated versus unauthenticated endpoints that the app contacts. If the majority of the app's requests are to secure API endpoints, use the authenticated <xref:System.Net.Http.HttpClient> instance as the default. Otherwise, register the unauthenticated <xref:System.Net.Http.HttpClient> instance as the default.
Note, that this is simply a way you can go with.
```

Please adjust the phrasing as necessary :)